docs/dev/e2e.md

```diff
@@ -3,18 +3,18 @@
 **Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
 
 - [Running E2E Tests](#running-e2e-tests)
-    - [Reference](#reference)
+  - [Reference](#reference)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
 # Running E2E Tests
 
 1. Ensure **operator-sdk** is installed and login to your OpenShift cluster as an admin user.
 
-1. Run the test using `make test-e2e`  command locally.
+1. Run the test using `make e2e-test-kind`  command locally.
 
     ```bash
-    make test-e2e
+    make e2e-test-kind
```

Comment:
I only get one question.

From my understand, does this mean that we need to use `quay.io/opencloud` because the e2e test is planning to integrate into CICD pipeline. 

But this doc and `make e2e-test-kind` is for both developer and CICD, right? Because developer could run `e2e` test and we have removed push command.

Reply:
even if e2e test is integrated into CICD, we wouldn't need quay.io/opencloudio. CICD would just use images from Artifactory

Reply:
OK, overall I think it is ok and convenient for us to use `quay.io/opencloud` for `e2e` test since it has the non-release tag and we will not push it into the `quay.io/opencloud`

Reply:
e2e test won't pull images from the remote image registry. It will just use the locally build image